docs/source/tutorial_optimization.rst

```diff
@@ -0,0 +1,900 @@
+in the exact same gradient but lacks computational speed.
+Moreover, the function ``output_d_mu`` returns a dict w.r.t. the parameter space as default.
+In order to use the output for :func:`~scipy.optimize.minimize` we thus use the ``return_array=True`` argument. 
+
+.. jupyter-execute::
+
+    def fom_gradient_of_functional(mu):
+        return fom.output_d_mu(fom.parameters.parse(mu), return_array=True, use_adjoint=True)
+    
+    opt_fom_minimization_data = {'num_evals': 0,
+                                'evaluations' : [],
+                                'evaluation_points': [],
+                                'time': np.inf}
+    tic = perf_counter()
+    opt_fom_result = minimize(partial(record_results, fom_objective_functional, opt_fom_minimization_data),
+                              initial_guess.to_numpy(),
+                              method='L-BFGS-B', 
+                              jac=fom_gradient_of_functional,
+                              bounds=(ranges, ranges),
+                              options={'ftol': 1e-15})
+    opt_fom_minimization_data['time'] = perf_counter()-tic
+    
+    # update the reference_mu because this is more accurate!
+    reference_mu = opt_fom_result.x
+
+
+
+
+.. jupyter-execute::
+
+    report(opt_fom_result, opt_fom_minimization_data)
+
+    
+
+
+With respect to the FOM result with finite differences, we see that we
+have a massive speed up by computing the gradient information properly.
+
+Optimizing using a gradient in ROM
+----------------------------------
+
+Obviously, we can also include the gradient of the ROM version of the
+output functional.
+
+
+.. jupyter-execute::
+
+    def rom_gradient_of_functional(mu):
+        return rom.output_d_mu(rom.parameters.parse(mu), return_array=True, use_adjoint=True)
+                
+    
+    opt_rom_minimization_data = {'num_evals': 0,
+                                 'evaluations' : [],
+                                 'evaluation_points': [],
+                                 'time': np.inf,
+                                 'offline_time': RB_greedy_data['time']}
+    
+    
+    tic = perf_counter()
+    opt_rom_result = minimize(partial(record_results, rom_objective_functional, opt_rom_minimization_data),
+                      initial_guess.to_numpy(),
+                      method='L-BFGS-B', 
+                      jac=rom_gradient_of_functional,
+                      bounds=(ranges, ranges),
+                      options={'ftol': 1e-15})
+    opt_rom_minimization_data['time'] = perf_counter()-tic
+    report(opt_rom_result, opt_rom_minimization_data, reference_mu)
+
+    
+
+
+The online phase is even slightly faster than before but the offline
+phase is obviously still the same as before. We also conclude that the
+ROM model eventually gives less speedup by using a better optimization
+method for the FOM and ROM. This is something that is a common issue for
+MOR methods in the context of optimization.
```

Comment:
I would remove the last sentence. Sounds a bit like 'don't use better optimization methods'.